kuzzleio/kuzzle-admin-console
# redirect to Login page if token expired

Fix #955 

## What does this PR do ?

As explained in the issue, as it we're unable to logout (to choose another environment) if token expired or if `auth.logout` action errored. 

### How should this be manually tested?

- Run a Kuzzle stack
- Run this PR Admin Console version
- Connect to your Kuzzle and create a first admin
- Find your Kuzzle docker container and stop it (`docker stop myContainer`)
- Restart it
- Now, it should redirect you to the current environment login page (and allow you to choose another environment)
